e2e/cypress/integration/enterprise/cloud/billing/after_subscription_spec.js

```diff
@@ -0,0 +1,402 @@
+
+        // # Visit Subscription page
+        cy.visit('/admin_console/billing/subscription');
+
+        // * Check for Subscription header
+        cy.contains('.admin-console__header', 'Subscription').should('be.visible');
+
+        // # Click Subscribe Now button
+        cy.contains('span', 'Subscribe Now').parent().click();
+
+        cy.intercept('POST', '/api/v4/cloud/payment/confirm').as('confirm');
+
+        cy.intercept('GET', '/api/v4/cloud/subscription').as('subscribe');
+
+        // # Enter card details
+        cy.uiGetPaymentCardInput().within(() => {
+            cy.get('[name="cardnumber"]').should('be.enabled').clear().type(billing.visa.cardNumber);
+            cy.get('[name="exp-date"]').should('be.enabled').clear().type(billing.visa.expDate);
+            cy.get('[name="cvc"]').should('be.enabled').clear().type(billing.visa.cvc);
+        });
+        cy.get('#input_name').clear().type('test name');
+        cy.findByText('Country').parent().find('.icon-chevron-down').click();
+        cy.findByText('Country').parent().find("input[type='text']").type('India{enter}');
+        cy.get('#input_address').clear().type('testaddress');
+        cy.get('#input_city').clear().type('testcity');
+        cy.get('#input_state').clear().type('teststate');
+        cy.get('#input_postalCode').clear().type('4444');
+
+        // # Click Subscribe button
+        cy.get('.RHS').find('button').should('be.enabled').click();
+
+        cy.wait(['@confirm', '@subscribe']);
+
+        // * Check for success message
+        cy.findByText('Great! You\'re now upgraded', {timeout: TIMEOUTS.TEN_SEC}).should('be.visible');
+
+        // * Check for starting date of the paid plan
+        cy.findByText(`Starting ${dayJs(new Date(currentDate.getFullYear(), currentDate.getMonth() + 1, 1)).format('MMM D, YYYY')} you will be charged based on the number of enabled users`).should('exist');
+
+        // # Click Let's go! button
+        cy.get('#payment_complete_header').find('button').should('be.enabled').click();
+    });
+
+    describe('System Console - Subscription section', () => {
+        it('MM-T4117 UI and Navigation check after subscription', () => {
+            navigateToBillingScreen('#billing\\/subscription', 'Subscription');
+
+            // * Check for non existence of trial tag
+            cy.contains('span', 'Trial').should('not.exist');
+
+            // * Check for '(Current Plan)' label
+            cy.contains('.PlanDetails__currentPlan>span', '(Current Plan)').should('be.visible');
+
+            // * Check for Paid label
+            cy.get('.paid>span').should('have.text', 'Paid');
+
+            // # Click on see Billing history link
+            cy.contains('span', 'See Billing History').parent().click();
+
+            // * Check for billing history navigation
+            cy.url().should('include', '/billing/billing_history');
+        });
+
+        it('MM-T4134 Downloading of invoice after subscription', () => {
```

Comment:
As we discussed here: https://community-daily.mattermost.com/core/pl/x3qrge3j77n6tyqtz69zd6jyny The invoice will not show as soon as they subscribe so I believe this test would be that there is no invoice here as soon as you subscribe (you can comment the test until we get the fix for this in).